presto-native-execution/presto_cpp/main/PrestoTask.cpp

```diff
@@ -485,10 +485,10 @@ protocol::TaskInfo PrestoTask::updateInfoLocked() {
   }
 
   if (taskStats.memoryReclaimCount > 0) {
-    taskRuntimeStats["memoryReclaimCount"].addValue(
+    taskRuntimeStats["taskMemoryReclaimCount"].addValue(
```

Comment:
Should we use constants for these and add comments explaining what these stats mean? Would be nice to also start documenting these. Thanks.

Reply:
Yeah, using constant will ease management. Will do in the followup with documentation. Thanks!